### ISSUE TITLE
hey guys new here quick question

### ISSUE THREAD

**Issue by Tecknoe** [kolton/d2bot-with-kolbot#3007]
hey everyone first time visiting the site i wanted to brush up on a little D2 and was looking for a maphack and i came across you guys, but currently the link is down for the maphack only is there anywhere else you guys have put it up?

also if not how would i configure the "bot" thats available for download just to use the maphack it comes with, or is it seperate and i can just run it on its own?

thanks in advance guys its been so long since ive used any bots // maphacks all that stuff.

**Comment by gtoilet** [comment]
![image](https://user-images.githubusercontent.com/11322067/114630966-b2d9c180-9c89-11eb-827f-66cc9ba30333.png)

**Comment by Tecknoe** [comment]
yeah i downloaded that, but i dont see the maphack just the bot, unless im blind.

**Comment by gtoilet** [comment]
install dependacies and get bot working then
1add
2edit
3set game location/entrypoint
4save
5start
WIN
![image](https://user-images.githubusercontent.com/11322067/115175715-8236c500-a099-11eb-84f5-f1ca5d77ca95.png)
